src/middleware/getFullFormSchema.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = (router) => (req, res, next) => {
+  router.formio.cache.loadCurrentForm(req, async (err, form) => {
+    if (!form) {
+      return next();
+    }
+
+    try {
+      await router.formio.util.Formio.createForm(form);
```

Comment:
Can we also assign a new property to the req object.

```
req.formInstance = await router.formio.util.Formio.createForm(form);
```

This way we could just use the formInstance in the new validator instead of having to reinstantiate it.